src/components/Room.tsx

```diff
@@ -58,12 +58,27 @@ function Room() {
     console.log("roomId:", window.location.href.split("/")[3]);
   }, [isConnected]);
 
+  async function shareRoom() {
```

Comment:
This should have Jest unit tests before it can be merged. If you haven't used jest before, im happy to work on this with you.

1. When the function is called, the `location.href` is copied to the clipboard
2. The `span` element is created. The Span has text 'URL Copied to Clipboard!'
3. That the span is removed after 500 ms

Reply:
Never done jest unit testing and could use the help!